app.json

```diff
@@ -14,6 +14,32 @@
   "scripts": {
     "postdeploy": "mix do ecto.migrate, run ./priv/repo/seeds.exs"
   },
+  "env": {
+    "API_HOST": {
+      "description": "The URL to which the Web application will connect for API requests",
+      "value": "https://heroku-app-name.herokuapp.com"
```

Comment:
The `API_HOST` in prod should probably be the same domain since both the API and the webapp are deployed on the same dyno. Same thing for `API_WS_HOST`.

Reply:
Yes, but what should be the values in `app.json`?

* `API_HOST` → `/`
* `API_WS_HOST` → `/`

We probably should rename `_HOST` to `_URL` in that case 😄

Reply:
Yes I think we should do that. The URL is only different in dev env when we want to develop using the ember-cli tooling.

I think that this will require changes in the webapp since we used the host as a string without the trailing slash.

Reply:
I changed the value to `this-heroku-app-name.herokuapp.com` for now. I think I could merge this (so we have a working `app.json` file) and improve the code later.

Sounds good?

Reply:
Yeah! Merge it!